src/main/java/be/seeseemelk/mockbukkit/inventory/meta/ItemMetaMock.java

```diff
@@ -276,6 +284,7 @@ public static ItemMetaMock deserialize(Map<String, Object> args) {
 	    serialMock.customModelData = (Integer) args.get("customModelData");
 	    serialMock.persistentDataContainer = (PersistentDataContainer) args.get("persistentDataContainer");
 	    serialMock.damage = (Integer) args.get("damage");
+        serialMock.repairCost = (Integer) args.get("repairCost");
```

Comment:
indentation

Reply:
Same as previous indentation "mistake" - this line is 8 spaces, which matches the rest of the file using spaces. The surrounding lines are 1 tab and 4 spaces.

Reply:
Weird, it should've been 4-spaces consistently.